other/travis/env-linux.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 CMAKE=cmake
-CMAKE_EXTRA_FLAGS=""
+CMAKE_EXTRA_FLAGS="-DFORMAT_TEST=ON"
```

Comment:
should we make this the default? 

As a developer, if I'm running tests, I'm about to commit (or already have) and if we don't plan to accept pulls that need to be formatted, shouldn't we tell them before they `get push`. 

If this does become the default. We should add an error message that tells devs how to disable this test while compiling so that we don't force them to go searching for the correct `disable_flag`